cocos2d/core/sprites/CCSpriteCanvasRenderCmd.js

```diff
@@ -55,12 +55,12 @@
         if (node._texture !== texture) {
             if (texture) {
                 if(texture.getHtmlElementObj() instanceof  HTMLImageElement)
-                    this._originalTexture = texture;
```

Comment:
This doesn't seems correct, you have made the following line controlled by the if statement while it's originally not in if